docs/fuel-101/transaction-lifecycle.mdx

```diff
@@ -5,4 +5,51 @@ category: Fuel 101
 
 # Transaction Lifecycle
 
-Fuel's transaction lifecycle explained.
+A transaction on the Fuel network goes through a complex series of steps before it is considered final:
+
+1. **Transaction Creation**: Someone sends a transaction to a node.
+2. **Pre-Validation**: The transaction goes through an extensive series of pre-checks, including UTXO validation.
+3. **Transaction Pool**: The transaction gets placed in the transaction pool.
+4. **Selection**: Transactions are selected from the pool.
+5. **Execution & Validation**: The transaction is validated by the FuelVM.
```

Comment:
@Dentosal can you clarify where/how it's different, or where I can find that info?

Reply:
The exact rules are defined here: https://github.com/FuelLabs/fuel-specs/blob/master/src/fuel-vm/index.md#predicate-verification

But what I meant is that it might make sense to mention here that predicates are executed as a part of tx validation process.